### PR TITLE
[Feature] August's documentation update

### PR DESCRIPTION
**Pull request by antoine4livre**

### CODE REVIEW

File: packages/docs/components/atoms/Figure/twig-api.md [hunk 1]
```diff
@@ -14,18 +14,21 @@ The  [`src`](#src), [`width`](#width) and [`height`](#height) parameters are req
...
 
 - Type: `string`
+- Required
 
 Configure the `src` attribute of the image.
 
 ### `width`
 
 - Type: `number`
+- Required
```
Comment (perruche) [inline packages/docs/components/atoms/Figure/twig-api.md:24]: **detail** width & height have default values `{% set width = width|default(100) %}`, they are not required

File: packages/docs/components/atoms/Cursor/index.md [hunk 2]
```diff
@@ -12,3 +12,26 @@ Use the cursor component to easily add a custom cursor to your project.
...
+    components: {
+      Cursor,
+    }
+  };
+}
+
+export default createApp(App);
+```
+```twig
+{% include '@ui/atoms/Cursor/Cursor.twig' %}
```
Comment (perruche) [inline packages/docs/components/atoms/Cursor/index.md:36]: **question** Should add the `only` attribute to the exemple ? To follow best-practice

File: packages/docs/components/atoms/LargeText/index.md [hunk 3]
```diff
@@ -1,13 +1,24 @@
-# LargeText (todo) <Badges :texts="badges" />
+# LargeText <Badges :texts="badges" />
 
 <script setup>
   import pkg from '@studiometa/ui/atoms/LargeText/package.json';
   const badges = [`v${pkg.version}`, 'JS', 'Twig'];
 </script>
 
+Use the large text component to easily add very nice looking title animation.
```
Comment (perruche) [inline packages/docs/components/atoms/LargeText/index.md:8]: **improvement** Could maybe have a clearer explanation. Use the LargeText component to implement a horizontal scrolling text on scroll. 
**unrelated**  This component have a name which isnt clear and descriptiive

File: packages/docs/components/atoms/LargeText/index.md [hunk 4]
```diff
@@ -1,13 +1,24 @@
...
+
 ## Table of content
 
 - [Examples](./examples.md)
 - [JS API](./js-api.md)
 - [Twig API](./twig-api.md)
```
Comment (perruche) [inline packages/docs/components/atoms/LargeText/index.md:14]: **unrelated to this PR/question** Is this way of structuring pages is the most helpful ? 
I'm wondering if it's good to have user going back and forth between multiple pages to view APi's and exemples. 
Any opinion on the subject @studiometa/core ?

File: packages/docs/components/atoms/LargeText/js-api.md [hunk 5]
```diff
@@ -2,4 +2,41 @@
...
+Remember to enable skew with the `skew` option.
+:::
+
+## Refs
+
+### `target`
+
+- Type: `HTMLElement`
+
+The `LargeText` component should have a ref corresponding to its inner `<span />` element.
```
Comment (perruche) [inline packages/docs/components/atoms/LargeText/js-api.md:42]: **detail** true that it is a span in the component provided by the package, but it's not mandatory when extending the component that the ref is a span. 
should it be phrase like : ... it's inner `HTMLElement` ?

File: packages/docs/components/molecules/Accordion/index.md [hunk 6]
```diff
@@ -9,3 +9,42 @@
...
+      Accordion,
+    },
+  };
+}
+
+export default createApp(App, document.body);
+```
+
+```twig{16}
+{% set items = [
```
Comment (perruche) [inline packages/docs/components/molecules/Accordion/index.md:34]: **question** Should we have a distinct section like `required props` for this type of component who do nothing if not provided with data ?

**Review by perruche — COMMENTED** [review]
So great to have the documentation moving forward 🚀

File: packages/docs/components/atoms/Cursor/index.md (hunk 2 above)
Comment (titouanmathis) [reply packages/docs/components/atoms/Cursor/index.md:36]: Yes!

File: packages/docs/components/atoms/LargeText/index.md (hunk 4 above)
Comment (titouanmathis) [reply packages/docs/components/atoms/LargeText/index.md:14]: We can add examples on the API pages, the examples pages can be a way to show more advanced or complex usages.

File: packages/docs/components/molecules/Accordion/index.md (hunk 6 above)
Comment (titouanmathis) [reply packages/docs/components/molecules/Accordion/index.md:34]: That could be interesting. Maybe a `*` symbol in the title could do the trick?

File: packages/docs/components/atoms/LargeText/index.md (hunk 3 above)
Comment (perruche) [reply packages/docs/components/atoms/LargeText/index.md:8]: yeah good name !

Comment (titouanmathis) [reply packages/docs/components/atoms/LargeText/index.md:8]: I suggest that we keep this one as is to not break projects using it. We can work on a new `Marquee` component with a more robust API.

File: packages/docs/components/atoms/LargeText/index.md (hunk 4 above)
Comment (antoine4livre) [reply packages/docs/components/atoms/LargeText/index.md:14]: Can we do that in the next documentation PR ?

File: packages/docs/components/molecules/Accordion/index.md (hunk 6 above)
Comment (antoine4livre) [reply packages/docs/components/molecules/Accordion/index.md:34]: Can we do that in a future PR ?

**Comment by antoine4livre** [comment]
@perruche @titouanmathis PR updated.

File: packages/docs/components/atoms/LargeText/index.md (hunk 4 above)
Comment (perruche) [reply packages/docs/components/atoms/LargeText/index.md:14]: sure, this is not blocking

File: packages/docs/components/molecules/Accordion/index.md (hunk 6 above)
Comment (perruche) [reply packages/docs/components/molecules/Accordion/index.md:34]: sure, this is not blocking

**Review by perruche — APPROVED** [review]

**Review by titouanmathis — APPROVED** [review]

**Pull request merged**